docs/docs/database.md

```diff
@@ -0,0 +1,49 @@
+# Database Project In-depth Guide
+
+We will be using PostgreSQL, a free, open-source database management system sucessor to UC Berkeley's Ingres Database but also support SQL language
```

Comment:
I would delete this sentence and replace with:

```
Our database is built using [Postgres](https://www.postgresql.org/).
```

The stuff about it being from UC Berkeley is fluff, and it's especially important to _not_ put fluff in the first sentence of a page of documentation, as it primes readers into believing there will be more fluff down the line and subtly/subconsciously encourages those readers to skim-read. We don't want people to think they have permission to skim-read!

Reply:
haha. Ok I will delete that.